docker-compose.test.yml

```
@@ -0,0 +1,96 @@
+      - test-network
+    profiles:
+      - test
+    command: go test -v -race -coverprofile=coverage.out ./...
+
+volumes:
+  mysql_test_data:
+    driver: local
+
+networks:
+  test-network:
+    driver: bridge 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)